### PR TITLE
Create release post 4.3 blog

### PR DESCRIPTION
**Pull request by iskyOS**
Release post 4.3, for publishing after December 15 release.
Changelog added from PMM: December 12
Highlights added from PM: December 12
Review/edits from PMM: December 13
Content review cycle: December 14

### Test
New blog post only.

### CODE REVIEW

File: content/blogposts/2022/release-4.3.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<img src="https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.3/Search%20Contexts.png alt="Search Contexts">
+
+<a href="https://docs.sourcegraph.com/code_search/how-to/search_contexts)" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<Badge link="/code-search" text="Code Search" color="cerise" size="small" />
+
+#### Webhooks for Repository and Permissions syncing 
+
+Sourcegraph predominatntly uses pull-based models for repository and permissions updates from the code host and webhooks have been hidden away in our external service config. But this can often create significant strain on code hosts and mean delays between syncs. In 4.3, webhooks have received a significant update making it easier for admins to configure webhooks for repository and permissions syncing including a new UI for creating, updating and deleting webhooks and improved handling of permissions webhooks. For now, our webhook updates are limited to GitHub customers.
```
Comment (pjlast) [inline content/blogposts/2022/release-4.3.md:60]:
```suggestion
Sourcegraph predominatntly uses pull-based models for repository and permissions updates from the code host and webhooks have been hidden away in our external service config. Polling code hosts to check for changes can often create significant strain on code hosts, and syncs take longer as the Sourcegraph instance grows. In 4.3, webhooks have received a significant update making it easier for admins to configure webhooks for repository and permissions syncing including a new UI for creating, updating and deleting webhooks and improved handling of permissions webhooks. What this means is that your Sourcegraph instance will mirror your code host changes much more closely, ensuring that users always see the latest versions of what they need to see. For now, our webhook updates are limited to GitHub customers. 
```

File: content/blogposts/2022/release-4.3.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+<a href="https://docs.sourcegraph.com/code_search/how-to/search_contexts)" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<Badge link="/code-search" text="Code Search" color="cerise" size="small" />
+
+#### Webhooks for Repository and Permissions syncing 
+
+Sourcegraph predominantly uses pull-based models for repository and permissions updates from the code host and webhooks have been hidden away in our external service config. Polling code hosts to check for changes can often create significant strain on code hosts, and syncs take longer as the Sourcegraph instance grows. In 4.3, webhooks have received a significant update making it easier for admins to configure webhooks for repository and permissions syncing including a new UI for creating, updating and deleting webhooks and improved handling of permissions webhooks. What this means is that your Sourcegraph instance will mirror your code host changes much more closely, ensuring that users always see the latest versions of what they need to see. For now, our webhook updates are limited to GitHub customers.
```
[the remark this answers is not in the document]
Comment (Joelkw) [reply content/blogposts/2022/release-4.3.md:85]: cc @alexAtSourcegraph this comment is safe to merge but if you'd like to edit further, feel free!

File: content/blogposts/2022/release-4.3.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+<a href="https://docs.sourcegraph.com/admin/config/webhooks" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+<br />
+<Badge text="Code Insights" link="/code-insights" color="green" size="small" />
+
+#### Code Insights are faster and even more reliable
```
Comment (iskyOS) [inline content/blogposts/2022/release-4.3.md:84]: @thisisvic Could you review this Code Insights highlight please?

File: content/blogposts/2022/release-4.3.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+tags: [blog, release]
+slug: "release/4.3"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.3/sourcegraph-4-3-hero.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.3/sourcegraph-4-3-hero.png
+changelogItems:
+  - description: "A \"copy path\" button has been added to file content, path, and symbol search results on hover or focus, next to the file path. The button copies the relative path of the file in the repo, in the same way as the \"copy path\" button in the file and repo pages."
+    url: https://github.com/sourcegraph/sourcegraph/pull/42721
+    category: Search
+  - description: "Unindexed search now uses the index for files that have not changed between the unindexed commit and the indexed commit. The result is faster unindexed search in general. If you are noticing issues you can disable by setting the feature flag `search-hybrid` to false."
```
Comment (iskyOS) [inline content/blogposts/2022/release-4.3.md:14]: FYI @lcolston08 for hybrid search. Did you want to add something for this one?

File: content/blogposts/2022/release-4.3.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+#### Instantly navigate repositories, files, and symbols by name with fuzzy finder 
+
+[Fuzzy finder](https://docs.sourcegraph.com/code_search/explanations/features) helps you quickly navigate to a repository, symbol, or file. To open the fuzzy finder, press 'Cmd+K' (macOS) or 'Ctrl+K' (Linux/Windows) from any page, and each tab has a dedicated shortcut:
+
+- Repos: Cmd+I (macOS), Ctrl+K (Linux/Windows)
+- Symbols: Cmd+O (macOS), Cmd+Shift+O (macOS Safari), Ctrl+O (Linux/Windows)
+- Files: Cmd+P (macOS), Ctrl+P (Linux/Windows)
+
+Fuzzy finder is an experimental feature, and we welcome your feedback and input.
```
Comment (amieroth) [inline content/blogposts/2022/release-4.3.md:41]: Do you want to link to where folks can go to give feedback?

**Review by amieroth — APPROVED** [review]
A couple comments and also made some small style edits that I already merged!

File: content/blogposts/2022/release-4.3.md (hunk 5 above)
Comment (iskyOS) [reply content/blogposts/2022/release-4.3.md:41]:
```suggestion
Fuzzy finder is an experimental feature, and we welcome your feedback and input. You can [tweet at us](https://twitter.com/sourcegraph) or [send us an email](mailto:feedback@sourcegraph.com).
```

File: content/blogposts/2022/release-4.3.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+<Badge text="Code Insights" link="/code-insights" color="green" size="small" />
+
+#### Insights are faster and even more reliable
+
+[Code Insights](https://docs.sourcegraph.com/code_insights) turns your codebase into a database, so you can track migrations, version spread, tooling, and any other custom metric from the source of truth—the code—directly. We made significant improvements to the backfiller that populates your data, making your insights faster and more reliable, including: 
+
+- Faster calculation times thanks to reduced contention in the work queue from separate insights
+- Graceful error handling to surface information to the user if there are any incomplete data points
+- Faster insight creation with more efficient background processing
+- Improved visibility into the state of background processes for easier troubleshooting and problem solving
```
[the remark this answers is not in the document]
Comment (iskyOS) [reply content/blogposts/2022/release-4.3.md:81]: I don't think this one is directly addressed in the mechanics guide, but looking at some older posts, we do have periods — so I'm going to adjust this one

**Review by amieroth — APPROVED** [review]
LGTM!

**Review by thisisvic — APPROVED** [review]

**Pull request merged**